cmd/avalanche.go

```diff
@@ -49,6 +50,7 @@ var (
 	remoteTenant        = kingpin.Flag("remote-tenant", "Tenant ID to include in remote_write send").Default("0").String()
```

Comment:
If we do what's in your PR, I would then remove `remoteTenant` flag and feature, as it will be possible via your flag. WDYT? I think breaking change here is fine, given it's a testing tool and not in a major version yet. WDYT?